lib/krane/deploy_task.rb

```diff
@@ -277,15 +277,18 @@ def validate_configuration(prune:)
     end
     measure_method(:validate_configuration)
 
-    def validate_resources(resources)
+    def validate_resources(resources, prune: false)
```

Comment:
Does the dry run behavior actually change with prune?

Reply:
It seems it really only changes the logging (for our concerns, at least). E.g. if `prune: true` and a resource in-cluster is not part of the provided manifests, the log output will show:

```
configmap/hello-cloud-configmap-data pruned (server dry run
```

Otherwise, it will simply be ignored. Since we swallow this part of the logs, I'm thinking it's best to drop the `prune` flag.